assets/js/environment.js

```diff
@@ -12,35 +12,22 @@ let streamData = []
 let streamVersionData = []
 
 function drawHealthChart(stream) {
-  const container = document.getElementById('healthTimeline')
-  const chart = new google.visualization.Timeline(container)
-  const dataTable = new google.visualization.DataTable()
-
-  dataTable.addColumn({ type: 'string', id: 'Status' })
-  dataTable.addColumn({ type: 'string', id: 'DummyLabel' })
-  dataTable.addColumn({ type: 'string', role: 'tooltip' })
-  dataTable.addColumn({ type: 'string', id: 'style', role: 'style' })
-  dataTable.addColumn({ type: 'date', id: 'Start' })
-  dataTable.addColumn({ type: 'date', id: 'End' })
-
-  // Start from last 25 (unless there is less than that)
-  const offset = stream.length <= 25 ? 0 : 25
-
-  for (let i = stream.length - offset; i < stream.length; i++) {
+  const traceUpX = []
+  const traceUpY = []
+  const traceUpText = []
+  const traceDownX = []
+  const traceDownY = []
+  const traceDownText = []
+  const traceUnknownX = []
+  const traceUnknownY = []
+  const traceUnknownText = []
+  var rangeStart
+  for (let i = 0; i < stream.length; i++) {
```

Comment:
A more functional way to do this would be `stream.forEach((streamItem, i) => {`

Then you have each item in the array without addressing by index/i.